packages/react-scripts/scripts/init.js

```diff
@@ -103,6 +103,13 @@ module.exports = function (
         'create-react-app'
       )} are no longer supported.`
     );
+    console.error(
```

Comment:
We actually know if the user is using Yarn or NPM here... but I also get that they could have globally installed with a different PM than they're using to init the current project.

Reply:
Would there be any way to detect what it's installed with?

Reply:
Not that I can think of - but I'm not an expert in that area. So I don't think anything needs to be actioned here.